Gems/AtomLyIntegration/AtomFont/Code/Source/AtomFont.cpp

```diff
@@ -327,12 +327,9 @@ namespace
 
 }
 
-AZ::AtomFont::AtomFont(ISystem* system)
-    : m_system(system)
-    , m_fonts()
+AZ::AtomFont::AtomFont([[maybe_unused]] ISystem* system)
+    : m_fonts()
```

Comment:
`: m_fonts()` can be removed as well, that just invokes the default constructor, which already occurs